doc/nrf/device_guides/working_with_nrf/nrf52/developing.rst

```diff
@@ -82,6 +82,45 @@ If you want to add SMP Service to advertising data, refer to the :ref:`zephyr:sm
 
 .. fota_upgrades_over_ble_additional_information_end
 
+FOTA updates are also supported when MCUboot is in Direct-XIP mode.
+In this mode, the MCUboot bootloader boots an image directly from a given slot, so the swap operation is not needed.
+It can be used either with or without the revert mechanism support.
+For more information about Direct-XIP mode and the revert mechanism support, go to the :doc:`mcuboot:index-ncs` page.
```

Comment:
Consider directly linking to the following section that describes the direct-xip mode:

https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/mcuboot/design.html#equal-slots-direct-xip

Reply:
@annwoj, is it possible to create such a reference?

Reply:
We could at least try linking the bootloader page. There's a directive for that but it requires further "manual" referencing, because, as far as I know, there's no way to link the actual section through "ref". So we could go with something along the lines: (...) go to the Equal slots (direct-xip) section on the :doc:`Bootloader <mcuboot:design.md>` page.